<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShoesDetailsMS</class>
 <widget class="QWidget" name="ShoesDetailsMS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>条件查询：</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>选择：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_condition">
          <item>
           <property name="text">
            <string>无条件</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>详情编号</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>商品编号</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>尺码</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>颜色</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>价格</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>图片数量</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>图片路径</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>库存</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>条件：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_condition"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pb_search">
          <property name="text">
           <string>搜索</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>条件结果：</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>相关操作：</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_10">
      <item row="2" column="0">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="1">
         <widget class="QPushButton" name="pb_delete">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pb_modify">
          <property name="text">
           <string>修改</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pb_cancel">
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="pb_ensure">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pb_add">
          <property name="text">
           <string>新增</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" rowspan="3">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>详情编号：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" rowspan="3">
         <widget class="QLineEdit" name="le_id"/>
        </item>
        <item row="0" column="3" rowspan="3">
         <widget class="QLineEdit" name="le_shoesid"/>
        </item>
        <item row="0" column="2" rowspan="3">
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>商品编号：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLabel" name="label_13">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>图片编号：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QLineEdit" name="le_photoid"/>
        </item>
        <item row="3" column="3">
         <widget class="QLineEdit" name="le_price"/>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_12">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>价格：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>颜色：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="le_color">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="9">
         <widget class="QLineEdit" name="le_size"/>
        </item>
        <item row="1" column="8">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>尺码：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QLabel" name="label_15">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>库存：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QLineEdit" name="le_stock">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
